src/server.ts

```diff
@@ -12,15 +12,19 @@ const SERVER_NAME: string = 'mediawiki-mcp-server';
 const SERVER_VERSION: string = packageInfo.version;
 
 export const createServer = (): McpServer => {
-	const server = new McpServer( {
-		name: SERVER_NAME,
-		version: SERVER_VERSION,
-		capabilities: {
-			resources: {
-				listChanged: true
+	const server = new McpServer(
+		{
+			name: SERVER_NAME,
+			version: SERVER_VERSION
+		},
+		{
+			capabilities: {
+				resources: {
+					listChanged: true
+				}
 			}
```

Comment:
@alistair3149 Was this working correctly before? I thought the structure changed now because CI failed when bumping to SDK 1.22 (https://github.com/ProfessionalWiki/MediaWiki-MCP-Server/pull/158). However, it looks like `capabilities` should've been in the second parameter for a long time now.